### DISCUSSION TITLE
Update in Win11: Defender Message

### DISCUSSION THREAD

**Discussion by nuki47** [valinet/ExplorerPatcher#3727] [General]
If I want to update my Toolbar, I got this message from Microsoft Defenender: HackTool:Win54/ExplorerPatcher!MTB

Actual version: 22621.3527.65.5

**Comment by pyrates999** [comment]
Duplicate of: https://github.com/valinet/ExplorerPatcher/issues/3228

Please close this.

MS just doesn't like EP and has added it to windows defender, even though it is not a virus and not a hack tool.

All builds of EP are done by github build servers against the source code in the repo.  Only binary is the reimplemented windows 10 taskbar and windows 10 start menu files.  There is no third party build servers.

This is a false positive.

See the release notes here for the powershell script you can run that excludes all the directories necessary in windows defender: https://github.com/valinet/ExplorerPatcher/releases

The powershell script you must run with elevated privileges.

Future updates to EP won't be flagged then.

You can also set windows defender to exclude the directory that you manually download EP to so you can install it without windows defender blocking it.

if you get the error: 0x800106ba, it means Windows Defender is not active. You may have another antivirus program active.
